src/commands/item.js

```diff
@@ -438,11 +439,32 @@ class Export extends Command {
 
       if (!path) return
 
-      const data =
-        yield call(db.seq, conn => mod.item.export(conn, [payload.id]))
+      const [resources, props] = yield select(state => {
```

Comment:
OK, let's leave that for the time being, but we'll have to utilize memoized selectors for this and seriously look at performance. A very real use case here is that you select 10k items and press Ctrl+C: it's OK for the export to take a while in such a case but we need to make sure we don't block the UI thread.

Reply:
I've tested export with 2400 items, was under 300ms according to the logs. Should be ok.
When the export will include images, I'll use the progress bar system, like with image import.